include/ginkgo/core/base/executor.hpp

```diff
@@ -1688,6 +1688,20 @@ class CudaExecutor : public detail::ExecutorBase<CudaExecutor>,
         return cusparse_handle_.get();
     }
 
+    /**
+     * Get the cusolver handle for this executor
+     *
+     * @return  the cusolver handle (cusolverDnContext*) for this executor
+     */
+    cusolverDnContext* get_dev_lapack_handle() const
+    {
+#if GKO_HAVE_LAPACK
+        return cusolver_handle_.get();
+#else
+        return NULL;
```

Comment:
```suggestion
        return nullptr;
```

Reply:
This is done in 54cfb58b266f3498b61fb21cfefd9eec8814407a